Wikipedia/Code/WMFSettingsViewController.m

```diff
@@ -489,13 +493,28 @@ - (NSInteger)tableView:(UITableView *)tableView
     return items.count;
 }
 
-- (nullable NSString *)tableView:(UITableView *)tableView
-         titleForHeaderInSection:(NSInteger)section {
-    NSString *header = [self.sections[section] getHeaderTitle];
-    if (header != nil) {
-        return header;
+- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section {
+    WMFTableHeaderFooterLabelView *header = (WMFTableHeaderFooterLabelView *)[self tableView:tableView viewForHeaderInSection:section];
+    if (header) {
+
+        if (@available(iOS 15.0, *)) {
+
+        } else {
+            //Hides odd content inset bug in iOS 13 & 14
+            if (section == 0) {
+                return 0;
+            }
+        }
```

Comment:
```suggestion
        if (@available(iOS 15.0, *) == false && (section == 0)) {
                //Hides odd content inset bug in iOS 13 & 14
                return 0;
        }
```

Reply:
Unfortunately with this code I see a warning that says `@available does not guard availability here; use if (@available) instead`. It does seem to work functionally the same, but I'm wary of putting this in with the warning.

Reply:
Thank you, good catch! (And sorry for not fully examining my suggestion.) We'll keep it as is.